tapir/coop/templates/coop/email/membershipresignation_confirmation_body.html

```diff
@@ -7,15 +7,15 @@
     <p>
         schade, dass Du uns verlassen möchtest. Hiermit bestätigen wir Dir die Kündigung deiner Mitgliedschaft bei SuperCoop Berlin eG.
     </p>
-    {% if resigned_member.willing_to_gift_shares_to_coop %}
+    {% if resigned_member.resignation_type == resignation_type_gift_to_coop %}
```

Comment:
Do I put as template tag the textchoices field from the model in lowercase with underscores? Is that the django syntax?

Reply:
The `resignation_type_gift_to_coop` variable is defined in the context that is sent to the template. In this case it's defined [here](https://github.com/SuperCoopBerlin/tapir/blob/409c7b63b00b929d97d792305687100b9a1f37ac/tapir/coop/emails/membershipresignation_confirmation_email.py#L43). then sent to the template [here](https://github.com/SuperCoopBerlin/tapir/blob/b5141312d9cae82e3741b49425b12602c0fe4de7/tapir/core/tapir_email_base.py#L73).